.travis.yml

```diff
@@ -1,16 +1,12 @@
 language: ruby
 rvm:
-  - 1.9.2
-  - 1.9.3
-  - jruby-19mode # JRuby in 1.9 mode
-  - rbx-19mode
+  - 2.4.5
+  - 2.5.3
+  - 2.6.0
+  - jruby-9.2.5.0 # JRuby in 2.5 mode
 services:
   - mongodb
 gemfile:
-  - gemfiles/mongoid2.gemfile
-  - gemfiles/mongoid3.gemfile
-matrix:
-  exclude:
-    # not supported by Mongoid 3
-    - rvm: 1.9.2
-      gemfile: gemfiles/mongoid3.gemfile
\ No newline at end of file
+  - gemfiles/mongoid5.gemfile
```

Comment:
vad betyder det att ha flera gemfiles? Är det beroende på andra installerade paket eller vad avgör vilken man ska köra med?

Reply:
Travis kommer att köra en testrunda med varje gemfile:

https://docs.travis-ci.com/user/languages/ruby/#testing-against-multiple-versions-of-dependencies

Reply:
aha, nice!